config/testgrids/kubernetes/sig-release/config.yaml

```diff
@@ -5,16 +5,14 @@ dashboard_groups:
   dashboard_names:
   - sig-release-master-blocking
   - sig-release-master-informing
+  - sig-release-1.27-blocking
+  - sig-release-1.27-informing
   - sig-release-1.26-blocking
   - sig-release-1.26-informing
   - sig-release-1.25-blocking
   - sig-release-1.25-informing
   - sig-release-1.24-blocking
   - sig-release-1.24-informing
-  - sig-release-1.23-blocking
-  - sig-release-1.23-informing
```

Comment:
Either keep the 1.23 blocking and informing dashboard, OR removed it entirely.

Reply:
I reverted this specific change and kept 1.23 dashboards